uiowa/uids
# Feature: LB feature maturity

In this pr: Resolves https://github.com/uiowa/uids/issues/300. 

# How to test

- [x] Resolves https://github.com/uiowa/uids/issues/298 - Adjust spacing under bold-headline--underline
-- View example https://uids.brand.uiowa.edu/branches/feature_lb_feature_maturity/components/detail/classes.html and verify that spacing between paragraph and headline exists. 
- [x] Resolves https://github.com/uiowa/uids/issues/299 - Lighten black bg brain background svg
-- Verify that opacity is at `.1` at https://github.com/uiowa/uids/blob/feature_lb_feature_maturity/src/assets/images/brain-pattern-black.svg?short_path=6e70b65
- [x] Resolves https://github.com/uiowa/uids/issues/289 - Add utility class for `<hr>`
-- View examples at the bottom of https://uids.brand.uiowa.edu/branches/feature_lb_feature_maturity/components/detail/classes.html. 
- [x] Resolves https://github.com/uiowa/uids/issues/224 - Refactor hero/banner into one component
-- View at https://uids.brand.uiowa.edu/branches/feature_lb_feature_maturity/components/detail/banner--default.html
- [x] Resolves https://github.com/uiowa/uids/issues/248 - Page title variant with background highlight
https://uids.brand.uiowa.edu/branches/feature_lb_feature_maturity/components/detail/banner--large_lower_left_highlighted_banner_caps.html and https://uids.brand.uiowa.edu/branches/feature_lb_feature_maturity/components/detail/banner--large_lower_left_highlighted_banner.html
- [x] Resolves https://github.com/uiowa/uids/issues/107 Slim crop variant of banner
-- https://uids.brand.uiowa.edu/branches/feature_lb_feature_maturity/components/detail/banner--small_centered_banner.html
- [x] Update Slider to use new banner
-- View Slider at https://uids.brand.uiowa.edu/branches/feature_lb_feature_maturity/components/detail/slider--default.html
- [x] Resolves https://github.com/uiowa/uids/issues/243 Play button centering
-- Code change  https://github.com/uiowa/uids/pull/301/commits/214b774f2f5a7b0011a2efe1e22a2a5f7821c7f7
- [x] Resolves https://github.com/uiowa/uids/issues/296 Card padding for horizontal variant bug
-- Go to https://uids.brand.uiowa.edu/branches/feature_lb_feature_maturity/components/detail/card--horizontal_border.html and view padding with and without an image. 
- [x] New Headline variants https://uids.brand.uiowa.edu/branches/feature_lb_feature_maturity/components/detail/headline.html
